src/main/java/com/uid2/shared/secure/azurecc/MaaTokenSignatureValidator.java

```diff
@@ -85,6 +85,7 @@ public MaaTokenPayload validate(String tokenString) throws AttestationException
 
         if(runtime != null){
             var runtimeDataBuilder = RuntimeData.builder();
+            runtimeDataBuilder.attestationUrl(tryGetField(runtime, "attestationUrl", String.class));
```

Comment:
From this MR we add user data to the userData prop: https://github.com/IABTechLab/uid2-attestation-azure/pull/15/files#diff-70c0be632c44d315449142e997600678038ec7c35f0a326dc92462c0e203b084R90

is there a corresponding change to the Azure attestation package?

Reply:
It's done https://github.com/IABTechLab/uid2-shared/pull/187/files#diff-2f5c6de8e97d4103b7e20f8ce1699cb6312997f2423d40d306ab1615783635dfR156

Reply:
Fixed